### DISCUSSION TITLE
Cannot compile due INTERNAL not defined

### DISCUSSION THREAD

**Discussion by kochikame** [ArminJo/Arduino-FrequencyDetector#4] [Q&A] [answered]
Cannot compile due INTERNAL not defined
Any hint?
Thanks in advance

**Comment by ArminJo** [comment]
Line 65 in your code?

**Comment by kochikame** [comment]
No, line 102
<img width="1342" alt="image" src="https://github.com/ArminJo/Arduino-FrequencyDetector/assets/578128/d55d7c4e-07e6-4791-8c9a-bfd3406214cc">

**Comment by ArminJo** [comment, marked as answer]
Please try INTERNAL1V1 instead of INTERNAL

**Comment by kochikame** [comment]
Great ! it compiles ok
Thank you for your help and for your code!
